devices/xiaomi/lumi_vibration_aq1.json

```diff
@@ -92,7 +92,7 @@
             "cl": "0x0000",
             "mf": "0x115F",
             "at": "0xFF0D",
-            "eval": "Item.val = 3 - Attr.val"
+            "eval": "Attr.val < 3 ? 3 - Attr.val : 2"
```

Comment:
Also needs to be assigned to the Item.val:

`Item.val = Attr.val < 3 ? 3 - Attr.val : 2`

Reply:
Hu, I have nothing to say